sql-db/hibernate/src/test/java/io/quarkus/qe/hibernate/BaseHibernateIT.java

```diff
@@ -30,6 +36,45 @@ public void shouldNotFailWithConstraints() {
         given().when().get("/items/count").then().body(is("1"));
     }
 
+    @Tag("https://github.com/quarkusio/quarkus/pull/46940")
+    @Test
+    public void testJsonEntityMappingWithJsonbUnremovable() {
+        String jsonPayload = "{\"testJsonEntity\": {\"field1\": \"value1\", \"field2\": 42}}";
+
+        String idString = given()
```

Comment:
Why don't we use `app.given()` here instead?

Reply:
Fixed